src/main/java/com/mewna/catnip/entity/interaction/ApplicationCommandInteractionDataOption.java

```diff
@@ -36,7 +36,8 @@
  * @since 12/10/20.
  */
 public interface ApplicationCommandInteractionDataOption extends HasName {
-    ApplicationCommandOptionType value();
+    ApplicationCommandOptionType type();
+    ApplicationCommandOptionChoice<?> value();
```

Comment:
It would be nice if this was specialised across all possible types, but not a huge issue for this PR